### ISSUE TITLE
Live feedback on subscription form

### ISSUE THREAD

**Issue by pgrenaud** [lanmomo/lanmomo-website#91]
Do live lookup using the API to check whether the username and email are available.

**Comment by S0yKaf** [comment]
just make sure the lookup is all done server side so people can't abuse the system and know the email of everyone who signed up.

ex:

``` javascript
function checkEmail(req, res) {
    if (emails.indexOf(req.body.email)) {
         res.json({emailTaken:true});
    } else {
    // stuff
    }
}
```

**Comment by pgrenaud** [comment]
Yep, no prob. I've already thought about that.

**Comment by jdupl** [comment]
closed by #92
